dtfs-central-api/src/repositories/utilisation-reports-repo/utilisation-report-sql.repo.ts

```diff
@@ -33,7 +33,7 @@ export const UtilisationReportRepo = SqlDbDataSource.getRepository(UtilisationRe
       }),
     );
 
-    const updatedReport = report.updatedWithUploadDetails({
+    const updatedReport = report.toReportWithUploadDetails({
```

Comment:
(i commented ditto where this method is actually defined rather than here!)

Reply:
Discussed: we are going to use the more OOP approach of mutating the state of the actual report rather than copying, updating and returning, meaning the method name is not `updateWithUploadDetails`